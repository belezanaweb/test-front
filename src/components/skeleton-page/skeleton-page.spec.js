import SkeletonPage from '.';

describe('SkeletonPage', () => {
  it('should match a snapshot', () => {
    const wrapper = getStyledWrapperJSON(SkeletonPage);

    expect(wrapper).toMatchSnapshot();
  });
});
