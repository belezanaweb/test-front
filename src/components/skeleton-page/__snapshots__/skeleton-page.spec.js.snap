// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonPage should match a snapshot 1`] = `
.c0 {
  padding: 1rem;
  position: fixed;
  width: 100%;
  height: calc(100% - 40px);
  background-color: #eee;
  box-sizing: border-box;
}

.c1 {
  display: inline-block;
  height: 5.625rem;
  width: 100%;
  background: linear-gradient(-90deg,#d8d6d6 0%,#fff 50%,#d8d6d6 100%);
  background-size: 400% 400%;
  -webkit-animation: pulse 1.2s ease-in-out infinite;
  animation: pulse 1.2s ease-in-out infinite;
  border-radius: 0.25rem;
  margin-bottom: 1rem;
}

.c1:last-child {
  height: 7.5rem;
  margin-top: 1rem;
}

<div
  className="c0"
>
  <div
    className="c1"
  />
  <div
    className="c1"
  />
  <div
    className="c1"
  />
  <div
    className="c1"
  />
</div>
`;
